obspy/core/inventory/response.py

```diff
@@ -1663,6 +1663,56 @@ def get_sacpz(self):
         paz = self.get_paz()
         return paz_to_sacpz_string(paz, self.instrument_sensitivity)
 
+    @staticmethod
+    def from_paz(zeros, poles, stage_sequence_number=1, stage_gain=1,
+                 stage_gain_frequency=1, input_units='COUNTS',
+                 output_units='M/S', normalization_frequency=1,
+                 pz_transfer_function_type='LAPLACE (RADIANS/SECOND)',
+                 normalization_factor=1.0, resource_id=None, resource_id2=None,
+                 name=None, input_units_description=None,
+                 output_units_description=None, description=None,
+                 decimation_input_sample_rate=None, decimation_factor=None,
```

Comment:
in my opinion: No need to set these - this is really just a helper function to construct the simplemost PAZ response.